unix/xserver/hw/vnc/vncInput.c

```diff
@@ -50,7 +50,7 @@ extern const unsigned int code_map_qnum_to_xorgevdev_len;
 extern const unsigned short code_map_qnum_to_xorgkbd[];
 extern const unsigned int code_map_qnum_to_xorgkbd_len;
 
-#define BUTTONS 7
+#define BUTTONS 11
```

Comment:
Let's use all the 16 buttons we have, so we can support devices with many buttons.

Reply:
The standard driver just uses `BTN_LABEL_PROP_BTN_UNKNOWN` for the extra ones, so that should be good enough for us as well.